rero_ils/modules/notifications/jsonschemas/notifications/notification-v0.0.1.json

```diff
@@ -24,11 +24,20 @@
       "type": "string",
       "minLength": 1
     },
-    "notification_date": {
+    "loan_pid": {
```

Comment:
replace by loan and remove patron, item, etc.

Reply:
done